physical/etcd/etcd3_test.go

```diff
@@ -26,7 +26,7 @@ func TestEtcd3Backend(t *testing.T) {
 		"username": "root",
 		"password": "insecure",
 
-		// Syncing adverticed client urls should be disabled since docker port mapping confuses the client.
+		// Syncing advertised client urls should be disabled since docker port mapping confuses the client.
 		"sync": "false",
 	}
```

Comment:
Should we add a test for list?

Reply:
There is one already, on line 44 of this file.